### PR TITLE
Update semantic color token names

### PR DESCRIPTION
**Pull request by jordanjones243**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** # (issue, if applicable)

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

* Update dependencies
* Refactor color tokens by isolating the color tokens and creating tier 3 tokens for repeatable rules

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: src/tokens.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+:host {
+  --ds-auro-table-background-color: var(--ds-color-container-secondary-default, $ds-color-container-secondary-default);
+  --ds-auro-table-border-color: var(--ds-color-border-divider-default, $ds-color-border-divider-default);
```
Comment (jason-capsule42) [inline src/tokens.scss:3]: These are missing the `#{...}` around the SCSS variable in order to make them compile correctly.

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: src/tokens.scss (hunk 1 above)
Comment (jordanjones243) [reply src/tokens.scss:3]: I don't think we need to do this in the `token.scss` file. It hasn't been done or pointed out in any of the other repos you or I have worked on.

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]
See comments

File: src/tokens.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
+@import "./../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables";
+
+:host {
+  --ds-auro-table-background-color: var(--ds-color-container-secondary-default, #{$ds-color-container-secondary-default});
```
Comment (jason-capsule42) [inline src/tokens.scss:4]: I realized we need a more precise token name here, This is not the background color of the whole table. We may want to support that in the future but currently we only have a color for the alternating rows.

```suggestion
  --ds-auro-table-row-alt-container-color: var(--ds-color-container-secondary-default, #{$ds-color-container-secondary-default});
```

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**